specification/quota/Quota.Management/client.tsp

```diff
@@ -65,3 +65,91 @@ using Microsoft.Quota;
 
 // Java-specific fixes for breaking changes
 @@clientName(ResponseETagHeader.eTag, "etag", "java");
+
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-operation" "customization"
+#suppress "@azure-tools/typespec-azure-core/documentation-required" "override"
+op currentQuotaLimitBasesCreateOrUpdate(
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "override"
+  apiVersion: string,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "override"
+  extensionProvider: "Microsoft.Quota",
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "override"
+  resourceName: string,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "override"
+  scope: string,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "override"
+  createQuotaRequest: CurrentQuotaLimitBase,
+): CurrentQuotaLimitBase;
+
+@@override(CurrentQuotaLimitBases.createOrUpdate,
```

Comment:
@kazrael2119 @v-jiaodi this is to mitigate the parmater ordering, do we have this breaking also?

Reply:
Yes，we need these also

Reply:
JS has this problem too?

So only .NET is immune to path ordering problem...